RecoEcal/EgammaCoreTools/src/EcalClusterTools.cc

```diff
@@ -1307,8 +1307,11 @@ float EcalClusterTools::getNormedIX(const DetId& id)
     if( id.subdetId()==EcalEndcap ) ix = EEDetId(id).ix();
     if( id.subdetId()==EcalShashlik ) ix = EKDetId(id).ix();
     if( ix != 0) {
-      int iXNorm  = ix - 50;
-      if(iXNorm<=0) iXNorm--;
+      float iXNorm; 
+      //in shashlik ix and iy run from 1 to 210
+      if( id.subdetId()==EcalShashlik ) iXNorm  = ix - 105.5;
+      else iXNorm = ix - 50.5;
+      //      if(iXNorm<=0) iXNorm--;
```

Comment:
It's not a matter of authorities. If I see a bug, I ask people to confirm that is a bug and I fix it. Even if it's not my business.
I'm sure it's right, anyway I don't care, I just want this pull request go as fast as possible, I don't have my laptop now, I will leave the wrong version for phase I asap, even if it's wrong.